ui/admin/app/controllers/scopes/scope/credential-stores/index.js

```diff
@@ -90,7 +90,7 @@ export default class ScopesScopeCredentialStoresIndexController extends Controll
   @notifySuccess('notifications.delete-success')
   async delete(credentialStore) {
     await credentialStore.destroyRecord();
-    await this.router.replaceWith('scopes.scope.credential-stores');
+    this.router.replaceWith('scopes.scope.credential-stores');
```

Comment:
hmm thx for fixing this, I guess best path forward is to not await transitions unless strictly necessary

Reply:
I think this was actually a mistake I had made in rebasing, I might have added back some awaits 🤔